textattack/loggers/csv_logger.py

```diff
@@ -20,17 +20,18 @@ def __init__(self, filename="results.csv", color_method="file"):
         self._flushed = True
 
     def log_attack_result(self, result):
-        if isinstance(result, FailedAttackResult):
-            return
         original_text, perturbed_text = result.diff_color(self.color_method)
+        result_type = result.__class__.__name__[:-12]
```

Comment:
haha. this is just to remove 'AttackResult' right? 

better: `result.class.__name__.replace('AttackResult', '')`

best: change the `GoalFunctionResultStatus` enum to map to a string (like `SUCCEEDED = 'succeeded'` instead of `SUCCEEDED = 0`) and then use `result.perturbed_result.status` (or something like that)